reactor-client/src/main/java/com/aerospike/client/reactor/IAerospikeReactorClient.java

```diff
@@ -719,11 +773,12 @@ Mono<KeyObject> execute(WritePolicy policy, Key key,
 	 * @param binName				bin name that data is indexed on
 	 * @param indexType				underlying data type of secondary index
 	 * @param indexCollectionType	index collection type
+	 * @param ctx					optional context to index on elements within a CDT
 	 * @throws AerospikeException	if index create fails
 	 */
 	Mono<Void> createIndex(Policy policy,
 						   String namespace, String setName, String indexName, String binName,
-			               IndexType indexType, IndexCollectionType indexCollectionType);
+			               IndexType indexType, IndexCollectionType indexCollectionType, CTX... ctx);
```

Comment:
It's a breaking change right? (I guess its ok to align with Aerospike Java Client that also contains a breaking change here) we need to make sure to state it in the release notes.